develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/01-adapting-to-liferay-7s-api.markdown

```diff
@@ -2,8 +2,8 @@
 
 +$$$
 
-**Important:** Liferay @ide@'s Code Upgrade Tool is not officially released and
-is unsupported. It's available in @ide@ 3.1 milestone versions only.
+**Important:** Liferay IDE's Code Upgrade Tool is not officially released and
```

Comment:
So I'm thinking we can putu the @ide@ back here since the same documentation will apply to both.  And once we have a GA release for studio it will be officially supported (as the same support level as all of our developer products, i.e. no SLAs)